ioctl/cmd/account/account.go

```diff
@@ -39,7 +47,7 @@ var (
 // AccountCmd represents the account command
 var AccountCmd = &cobra.Command{
 	Use:   "account",
-	Short: "Manage accounts of IoTeX blockchain",
+	Short: accountCmdShorts[config.Language],
```

Comment:
```
    type Language int
    const (
        English Language = iota
        Chinese
    )

    func SelectByLang(translations map[Language]string, lang Language) {
        if tsl, ok := translations[lang]; ok {
            return tsl
        }

        // Assumption: English should always be provided
        return translations[English]
    }
```

Then we can use it in this way
`Short: SelectByLang(accountCmdShorts, config.Language)`